legacyCore/src/main/res/layout-sw300dp/title_text.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This should be a style

Reply:
oof I missed those in the the seperate layout folder, made into a style and removed the layout

Reply:
made into a style